example/MODULE.bazel

```diff
@@ -1,30 +1,23 @@
 "Bazel dependencies"
 
-bazel_dep(name = "aspect_rules_lint", version = "0.0.0", dev_dependency = True)
```

Comment:
note: dev_dependency is technically better here, but I just want to reduce the size of the example to make it less intimidating for casual readers